packages/loader/container-loader/src/containerContext.ts

```diff
@@ -145,7 +145,7 @@ export class ContainerContext implements IContainerContext {
         return this._baseSnapshot;
     }
 
-    public get storage(): IDocumentStorageService | undefined | null {
```

Comment:
>| [](start = 62, length = 1)

why do we need to add a null state here?

Reply:
I have no clue why null was here, it was not used, so I deleted it (your comment suggests that I'm adding it, which is not the case)